src/webhooks/webhook.ts

```diff
@@ -0,0 +1,61 @@
+import { runCommand } from '@lib/runCommand.js';
+import { linkMergeRequest, linkBranch, referenceToRecordFromTitle, linkPullRequestToRecord } from '../lib/fields.js';
+
+aha.on('webhook', async ({ headers, payload }) => {
+  const event = payload.eventType;
+
+  console.log(`Received webhook '${event}' ${payload.event_type || ''}`);
+
+  switch (event) {
+    case 'git.branch.created':
```

Comment:
@sorza-jimmy How do you register this service hook in Azure? I can only generate a `git.push` event for new branches.

Reply:
@trydionel  Sorry. I missed that. I updated